lib/defaults.js

```diff
@@ -33,10 +33,10 @@ module.exports = {
     },
   ],
   groupTypeId: "sap.cds:service",
-  packages: function getPackageData (name, policyLevel,capNamespace) {
+  packages: function getPackageData (name, policyLevel,namespace) {
     function createPackage(name, tag) {
       return {
-        ordId: `${regexWithRemoval(name)}:package:${capNamespace}${tag}`,
+        ordId: `${regexWithRemoval(name)}:package:${namespace}${tag}`,
```

Comment:
Is this package ordId valid? `<namespace>:<conceptName>:<resourceName>:[v<majorVersion>]`, the last part is `${namespace}${tag}`?

I also suggest provides variables as context, and extract it into function, for example:
```suggestion
        ordId: getPackageOrdID(),
       ...
       function getPackageOrdID() {
         const namespace = namesapce;
         const conceptName = package;
         const resourceName = sourceName;
         const version = version:
         return `${namespace}:${conceptName}:${resourceName}:${version}:`
       }
```

Reply:
See above. You need to use the globally defined ORD namespace , not `regexWithRemoval(name)`

Reply:
Btw. you have typo with "namesapce"